### PR TITLE
User guide removed

### PR DESCRIPTION
**Pull request by czgdp1807**
#### References to other Issues or PRs

https://github.com/sympy/sympy/pull/2807
https://github.com/sympy/sympy/pull/10330

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by czgdp1807** [comment]
ping @asmeurer @oscarbenjamin @certik

**Comment by certik** [comment]
@asmeurer if you think this guide does not apply anymore then let's remove it.

File: doc/src/guide.rst [hunk 1]
```diff
@@ -1,593 +0,0 @@
...
-X):
-
-.. parsed-literal::
-
-    $ :input:`winpdb t.py`
-    y + 2*x
-
-and a winpdb window will popup, I move to the next line using F6:
-
-.. image:: pics/winpdb1.png
```
Comment (sylee957) [inline doc/src/guide.rst:57]: I see there are some associated images for this tutorial. If they are not used anymore they should be removed manually too

**Comment by Sc0rpi0n101** [comment]
If this plans to finish #2807, I guess #10330 can also be closed

File: doc/src/guide.rst (hunk 1 above)
Comment (czgdp1807) [reply doc/src/guide.rst:57]: Done.

**Comment by czgdp1807** [comment]
Please let me know if any change is to be made. @certik @asmeurer Please merge whenever it appears to be ready.

**Comment by asmeurer** [comment]
I still think we should remove it. There were some objections on the other PR, but I still maintain that there isn't anything useful in this document. If we really want to salvage anything, we should put it on the wiki somewhere.

**Comment by czgdp1807** [comment]
@asmeurer Can we merge this?

**Pull request merged**